packages/dx-react-scheduler/src/plugins/drag-drop-provider.jsx

```diff
@@ -162,35 +143,22 @@ export class DragDropProvider extends React.PureComponent {
       if (targetType === VERTICAL_TYPE) {
```

Comment:
It seems that the code computing this.allDayRects and this.timeTableRects can be extracted to separate function too. It will require 5-6 parameters but calculateBoundaries will be easier to read